app/config/storage/table.js

```diff
@@ -0,0 +1,25 @@
+const Joi = require('joi')
+
+const schema = Joi.object({
+  useConnectionString: Joi.bool().default(false),
+  connectionString: Joi.string().when('useConnectionString', { is: true, then: Joi.required(), otherwise: Joi.allow('').optional() }),
+  account: Joi.string().required(),
```

Comment:
A very picky comment but perhaps better to keep consistent naming with blob.js where 'account' is actually 'storageAccount'

Reply:
Good point - just pushed an update.